utils/gitlab_api.py

```diff
@@ -155,6 +180,15 @@ def close_issue(self, issue):
         issue.state_event = 'close'
         issue.save()
 
+        
+    def get_user(self, username):
+        user = self.gl.users.list(search=username)
+        if len(user) == 0:
```

Comment:
same here. if a user is not found we should be less forgiving imo

Reply:
I am not sure the solution for this one. If a user is in app-interface but does not have a gitlab account I don't really think this is the integration that should be making those users. I don't think that is in the scope of the integration because it is more about groups.

Do you have a suggestion for a less forgiving way to deal with this if it happens? Possibly sending an email to the redhat user telling them to make a gitlab account similar to the github_users integration. Or if you think the integration should be making accounts it should probably notify the user that a account was created for them and what their username and pass is.

Could be a new integration that checks what users should have gitlab accounts and creates/emails them if it doesn't fit with this one.

Reply:
i just meant that we should do more then just log. maybe exit with an error in this case, or to keep consistency with previous discussion, log an error and continue?